mysql-test/t/percona_innodb_fake_changes.test

```diff
@@ -368,5 +372,53 @@ COMMIT;
 
 DROP TABLE t1, t3;
 
+#-------------------------------------------------------------------------------
+#
+# Scenario will try to insert data that will cause split of a node something
+# which will not happen in real if fake_changes = 1.
+#
+
+# clustered index split
+use test;
+create table t1 (i int, j blob, primary key pk(i)) engine=innodb;
+insert into t1 values (1, repeat('a', 8000));
+insert into t1 values (2, repeat('b', 8000));
+set session innodb_fake_changes = 1;
+--error ER_ERROR_DURING_COMMIT
+insert into t1 values (3, repeat('c', 8000));
```

Comment:
So based on our discussion on IM initially here is how I looked at it.
-----

We have 2 options:

a. Add scenario to existing TC (as done currently). TC will work in same way ir-respective of the page-size except that in case of 16K it would trigger a pessimistic insert case (for other 4K/8K it will try to use external page storage). So even if we port to run our suites with different page-size this test-case will be covered to test it out and will pass with all pages size. (I have verified this)

b. Leave the existing TC as is and add another test-case say percona_bug_1410410.test and restrict it to run only for 16K. But this ultimately will achieve the same goal as #a infact the test-scenario will never run for 4K/8K as we have explicit block for 16K only.